akka-stream/src/main/scala/akka/stream/scaladsl/Restart.scala

```diff
@@ -464,7 +464,7 @@ private abstract class RestartWithBackoffLogic[S <: Shape](
         if (finishing || maxRestartsReached()) {
           fail(out, ex)
         } else {
-          log.error(ex, "Restarting graph due to failure")
+          log.warning(ex, "Restarting graph due to failure")
```

Comment:
Build will fail until I pull in https://github.com/akka/akka/pull/25033.

Reply:
Due to the hazzle of adding that in a binary compatible way we decided to not do that. Please change this to:
```
if (log.isWarningEnabled)
  log.warning("Restarting graph due to failure: {}", Logging.stackTraceFor(e))
```